classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
```

Comment:
`students` here should be the instance variable `@students` that you initialized in the constructor above. Kindly use the `@students` instead.